### ISSUE TITLE
Update-CWServiceTicket -DueDate

### ISSUE THREAD

**Issue by vedeht** [sgtoj/PSConnectWise#37]
Can the update-cwserviceticket update the due date on a ticket?

**Comment by sgtoj** [comment]
Have been on vacation for past week plus. I will work on this and #36 as soon as I finished catching up with my other work duties.

**Comment by vedeht** [comment]
Thank you!

Devin Henderson
 @vedeht

On Oct 3, 2016, at 1:25 PM, Brian <notifications@github.com> wrote:

Have been on vacation for past week plus. I will work on this and #36
<https://github.com/sgtoj/PSConnectWise/issues/36> as soon as I finished
catching up with my other work duties.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/sgtoj/PSConnectWise/issues/37#issuecomment-251169017>,
or mute the thread

**Comment by sgtoj** [comment]
Just wanted to let you know I haven't forgotten about this and your other requests (#36 and #31). Right now, my primary focus is getting #21 resolved though. I (better yet, my company/department) uses this module extensively for work. How exception-handling was original implemented has caused problems tracking down issues when being consumed by other scripts. In other words, it important to resolve that issue.

Along with resolving this that issue, I have to become AWS Developer certify, develop a new REST API service for my department, and along couple other tasks. This basically means, it will take longer than I expected because I will be using my free time (e.g. weekends) to work on your requests. **But it will be completed.**

**Comment by vedeht** [comment]
Thanks for the update!  You should see what I did with your scripts!  I
think you would laugh and be amazed!

Devin Henderson


On Oct 6, 2016, at 7:29 AM, Brian <notifications@github.com> wrote:

Just wanted to let you know I haven't forgotten about this and your other
requests (#36 <https://github.com/sgtoj/PSConnectWise/issues/36> and #31
<https://github.com/sgtoj/PSConnectWise/issues/31>). Right now, my primary
focus is getting #21 <https://github.com/sgtoj/PSConnectWise/issues/21>
resolved though. I (better yet, my company/department) uses this module
extensively for work. How exception-handling was original implemented has
caused problems tracking down issues when being consumed by other scripts.
In other words, it important to resolve that issue.

Along with resolving this that issue, I have to become AWS Developer
certify, develop a new REST API service for my department, and along couple
other tasks. This basically means, it will take longer than I expected
because I will be using my free time (e.g. weekends) to work on your
requests. *But it will be completed.*

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/sgtoj/PSConnectWise/issues/37#issuecomment-251936266>,
or mute the thread

**Comment by vedeht** [comment]
Any update on adding additional fields to be "Updated" from update-cwserviceticket?  I have some powershell scripts that check for certain fields, etc settings and populate some data in fields on a ticket.  I can only update the status of a ticket or write some notes on a ticket.

**Comment by sgtoj** [comment]
@vedeht With the quarter, I finished the big projects I was tasked with at work. Although I will have more task for this quarter, I will create a task to work on this project for my next 2-week sprint.
